patches/ui-webui-resources-tools-bundle_js.gni.patch

```diff
@@ -0,0 +1,20 @@
+diff --git a/ui/webui/resources/tools/bundle_js.gni b/ui/webui/resources/tools/bundle_js.gni
+index 56dc603dc7ed6781af6961e274ec466663d9885d..3d338d333c508f05cf072c8c0d4e72512d1ba5c0 100644
+--- a/ui/webui/resources/tools/bundle_js.gni
++++ b/ui/webui/resources/tools/bundle_js.gni
+@@ -67,6 +67,7 @@ template("bundle_js") {
+         rebase_path("$root_gen_dir/ui/webui/resources/tsc/", root_build_dir)
+     is_chrome_untrusted =
+         string_replace(invoker.host, "chrome-untrusted://", "") != invoker.host
++    brave_resources_path = rebase_path("$root_gen_dir/brave/ui/webui/resources/tsc/", root_build_dir)
```

Comment:
this should go on the same line as https://github.com/brave/brave-core/pull/18388/commits/d59b0636edd0ce39d2f82f115df4daa21d1883c0#diff-d3ec102d1b14364a2e583ead168a85b3d92f9f37e738793211db868e4c4a8385R17

Reply:
but can we modify the node template instead so we only do this in one place?